hieradata_aws/staging.yaml

```diff
@@ -224,6 +224,7 @@ licensify::apps::licensify_feed::environment: 'staging'
 monitoring::checks::aws_origin_domain: "staging.govuk.digital"
 monitoring::checks::datagovuk_publish::ensure: 'present'
 monitoring::checks::datagovuk_publish::host: 'publish-data-beta-staging.cloudapps.digital'
+monitoring::checks::datagovuk_publish::protocol: 'https'
```

Comment:
As you don't need to configure different values for this, I would suggest just specifying it in the manifest, rather than through a parameter.